README.md

```diff
@@ -197,6 +197,32 @@ However, here are some rules that we'd like to emphasize since they are not visi
 |Mentor                  | Kannupriya Kalra ([LinkedIn](https://www.linkedin.com/in/kannupriyakalra/), Email: [kannupriyakalra@gmail.com](mailto:kannupriyakalra@gmail.com)) |
 |Co-mentor               | Rory Graves ([LinkedIn](https://www.linkedin.com/in/roryjgraves/), Email: [rory.graves@fieldmark.co.uk](mailto:rory.graves@fieldmark.co.uk)) |
 
+### Difflicious UI
+
+| Title                    | Difflicious UI                                                                                                                                                                                                           |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Link to Project          | https://github.com/jatcwang/difflicious                                                                                                                                                                                  |
+| Brief Description        | Implement a Web UI for Difflicious which allows users to explore diffs (test failures) reported by Difflicious. The UI should allow the user to expand and minimize section of the diff output in an interactive manner. |
+| Expected Outcome         | Working UI and documentation                                                                                                                                                                                             |
+| Prerequisites            | Good working knowledge of Scala and web technologies (Javascript, CSS, HTML). The project will be implemented in Scala.js (most likely with [Laminar](https://laminar.dev/)                                              |
+| Expected Difficulty      | Hard                                                                                                                                                                                                                     |
+| Expected Time Commitment | Large project - 350 hours                                                                                                                                                                                                |
```

Comment:
Fresh difflicious user here 😃
Personally I would have thought this is a project for two weeks with a little AI assistance, so I guess I'm missing some complexities. Could you explain what you think is so hard/time-consuming here?

Reply:
Hi @nightscape, and thanks for the comment. I need to merge this PR now because the application deadline is today.

@jatcwang feel free to continue the discussion here. You can submit a new PR anytime if you need to edit anything. Nothing is written in stone.

Reply:
Thanks for the comment @nightscape. I made a large project because the project requires quite a wide range of skills (UI design, Scala-js, FRP, frontend, HTTP API) and the student might need time to upskill.
For the tech chosen, AI won't be able to help as much IMO due to sample size.